Diagrams/Tetris flow.md

```diff
@@ -0,0 +1,52 @@
+graph TD
```

Comment:
All looks to be good. Curious if GitHub will render this in .md or if we’ll have to dump to html or pdf.